repo/packages/G/geoserver/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion" : "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name" : "geoserver",
+  "version" : "2.10.0-1.1.0",
```

Comment:
yes, I suggest 0.0.1 because that is the convention for the packaging version in Universe. 0.0.1 because this is the first Geoserver package in Universe. Any update would increment this version number. But, it is upto the package submitter so you can keep what you have now. 👍